staging/src/k8s.io/client-go/tools/record/event.go

```diff
@@ -275,7 +275,7 @@ func recordEvent(sink EventSink, event *v1.Event, patch []byte, updateExistingEv
 func (e *eventBroadcasterImpl) StartLogging(logf func(format string, args ...interface{})) watch.Interface {
 	return e.StartEventWatcher(
 		func(e *v1.Event) {
-			logf("Event(%#v): type: '%v' reason: '%v' %v", e.InvolvedObject, e.Type, e.Reason, e.Message)
+			klog.InfoS("Event occurred", "object", e.InvolvedObject, "type", e.Type, "reason", e.Reason, "message", e.Message)
```

Comment:
```suggestion
			klog.InfoS("Event occurred", "object", klog.KObj(e.InvolvedObject), "kind": e.InvolvedObject.Kind, "apiVersion": e.InvolvedObject.APIVersion, "type", e.Type, "reason", e.Reason, "message", e.Message)
```
Please apply instructions for logging k8s objects from https://github.com/kubernetes/community/blob/fc2bfe4c10e901d5d214697a6717c0dc98e6fbd7/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#use-klogkobj-and-klogkref-for-kubernetes-objects

Reply:
Didn't check what type is e.InvolvedObject, so if not k8s object (implements `GetName` and `GetNamespace`), you can use `klog.KRef(e.InvolvedObject.Namespace, e.InvolvedObject.Name)`

Reply:
Thank you for your review.
I could not use `klog.KObj`, so I use `klog.KRef` to apply the instruction document.